### PR TITLE
add vikalp-tomer

### PR DESCRIPTION
**Pull request by vikalp-tomer**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2111"></a>

### CODE REVIEW

**Review by AbbeyIT — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by vikalp-tomer** [comment]
thanks

File: public/data/vikalp-tomer.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "icon": "linkedin"
+      },
+      {
+        "name": "Check out my GitHub",
+        "url": "https://github.com/vikalp-tomer",
+        "icon": "github"
+      }
+    ]
+  }
+
```
Comment (eddiejaoude) [inline public/data/vikalp-tomer.json:19]: File is not finished with a blank line, not a blocker, it is marked with red sign here by GitHub - just to make you aware for next time

Comment (vikalp-tomer) [reply public/data/vikalp-tomer.json:19]: Thanks. Will remember from next time.

**Pull request merged**